releases/release-1.22/release-notes/maps/pr-100292-map.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 100292
+releasenote:
+  text: Updated pause image to version 3.5, which now runs per default as pseudo user
+    and group `65535:65535`. This does not have any effect on remote container runtimes
+    like CRI-O and containerd, which setup the pod sandbox user and group on their
+    own.
```

Comment:
i think when there are backticks, the block needs to be quoted / escaped with `-|`

```suggestion
  text: 'Updated pause image to version 3.5, which now runs per default as pseudo user
    and group `65535:65535`. This does not have any effect on remote container runtimes
    like CRI-O and containerd, which setup the pod sandbox user and group on their
    own.'
```

an easy way to check if YAML is valid is to use some YAML parser and see if it panics 😄

Reply:
Thanks for helping with review. I was being lazy and used some online yaml validator. And it does not complain 🤔

Reply:
heh you're right, ignore my comment then! 😄 I think I misremembered this and confused it with something else.

Reply:
I tried yq / yamllint, both didn't complain. Did I miss anything? 😳

Reply:
Sorry didn't see your earlier comments. Thanks ^^